pkg/kubelet/metrics/metrics.go

```diff
@@ -36,6 +36,7 @@ const (
 	PodWorkerDurationKey               = "pod_worker_duration_seconds"
 	PodStartDurationKey                = "pod_start_duration_seconds"
 	PodStartSLIDurationKey             = "pod_start_sli_duration_seconds"
+	ContainerStartDurationKey          = "container_start_duration_seconds"
```

Comment:
There is already a `container_start_time_seconds' in `/metrics/resource'.

Reply:
https://github.com/kubernetes/kubernetes/blob/4d105eb1ce80edf7ccc2ee4e2a69ab44053185b6/pkg/kubelet/metrics/collectors/resource_metrics.go#L107-L112

Reply:
container_start_time_seconds is the epoch timestamp at which the container starts. The metric added in this PR is the duration/latency of container startup. They are different.